src/components/ui/primary-button.tsx

```
@@ -0,0 +1,61 @@
+          'w-full px-3 py-2 text-sm font-semibold text-white bg-blue-600 rounded-md',
+          'hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-600 focus:ring-offset-2',
+          'disabled:opacity-50 disabled:cursor-not-allowed',
+          'transition-colors duration-200',
+          'min-h-[44px]', // Ensure 44px minimum touch target
+          className
+        )}
+        disabled={disabled || isLoading}
+        {...props}
+      >
+        {isLoading ? (
+          <span className="flex items-center justify-center">
```

ESLint

[error] Replace `"flex·items-center·justify-center"` with `'flex·items-center·justify-center'`

(prettier/prettier)